schema/me/consignments/add_asset_to_submission_mutation.js

```diff
@@ -0,0 +1,32 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { AssetType } from "./asset"
+
+export default mutationWithClientMutationId({
+  name: "AddAssetToConsignmentSubmission",
+  description: "Attach an impulse asset to a consignment submission",
+  inputFields: {
+    asset_type: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The type of the asset, currently only 'image' is supported",
```

Comment:
We'll probably be adding more asset types soon, so I don't think this qualifier is necessary.